.github/workflows/mobile.yml

```diff
@@ -19,18 +19,24 @@ jobs:
             npm install
             npm run prebuild:electron
 
-        - name: Build project artifacts (iOS)
+        - name: Install project platform (iOS)
           run: |
             cd cordova
             npm install
             npm run install:ios
+            npm run prebuild:ios
+
+        - name: Build artifacts
+          run: |
+            cd cordova
             npm run build:ios
 
         - name: Upload artifacts
-          uses: actions/upload-artifact@v2
+          uses: actions/upload-artifact@v4
           with:
-            name: Solar.app
-            path: /Users/runner/work/mtl_solar/mtl_solar/cordova/platforms/ios/build/emulator/Solar.app
+            name: MTL_Solar_ios.app
+            path: ${{ github.workspace }}/cordova/platforms/ios/build/emulator/Montelibero Solar.app
```

Comment:
Should we avoid having namespace in the filename?

Reply:
What do you mean by "namespace"?